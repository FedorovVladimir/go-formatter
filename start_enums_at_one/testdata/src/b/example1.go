package main

type Operation int

const (
	Add Operation = iota // want "start_enums_at_one"
	Subtract
	Multiply
)
