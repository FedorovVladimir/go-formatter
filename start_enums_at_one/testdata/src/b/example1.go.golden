package main

type Operation int

const (
	Add Operation = iota + 1 // want "start_enums_at_one"
	Subtract
	Multiply
)
